ambry-frontend/src/main/java/com.github.ambry.frontend/GetSignedUrlHandler.java

```diff
@@ -207,12 +207,21 @@ public void onCompletion(Void result, Exception exception) {
           restResponseChannel.setHeader(RestUtils.Headers.DATE, new GregorianCalendar().getTime());
           restResponseChannel.setHeader(RestUtils.Headers.SIGNED_URL, signedUrl);
           restResponseChannel.setHeader(RestUtils.Headers.CONTENT_LENGTH, 0);
+          final long processResponseStartTimeMs = SystemTime.getInstance().milliseconds();
+          securityService.processResponse(restRequest, restResponseChannel, null,
+              (voidResult, processResponseException) -> {
```

Comment:
minor: instead of `voidResult` you could use `processResponseResult` as the name

Reply:
Changed